docs/tutorial/agent-on-docker.md

```diff
@@ -0,0 +1,155 @@
+# Agent on Docker
+
+Vald is designed and implemented based on Cloud-Native architecture.
+However, there may be cases who want to use only Vald Agent without kubernetes.
+
+This article will show you how to deploy and run the Vald Agent on Docker.
+Fashion-mnist is used as an example of a dataset as same as [Get Stared](../tutorial/get-started.md)
+
+## Requirements
+
+- docker: v19.0 ~
+- go: v1.14 ~
+- libhdf5 (_only required for this tutorial._)
+
+
+Hdf5 is required for this tutorial. If hdf5 is not installed, please install [hdf5](https://www.hdfgroup.org/).
+<details><summary>[Optional] Install hdf5</summary><br>
+
+```bash
+# yum
+yum install -y hdf5-devel
+
+# apt
+apt-get install libhdf5-serial-dev
+
+# homebrew
+brew install hdf5
+```
+</details>
+
+## Deploy
+
+This chapter will show you how to deploy Vald Agent on docker.<br>
+This chapter will use NGT for the core method of Vald Agent.
+
+1. Clone the vdaas/vald repository
+
+    ```bash
+    git clone https://github.com/vdaas/vald.git
+    ```
+
+1. Create `config.yaml`
+
+    The configuration of Vald agent for docker is set using `config.yaml`<br>
+    You also check [the sample](https://github.com/vdaas/vald/blob/master/cmd/agent/core/ngt/sample.yaml).
+
+    ```bash
```

Comment:
is it better to use 
```
cat << EOF > /tmp/config.yaml
....
EOF
```
here and change the docker run command to specify the config file path?

Reply:
I think we don't have to specify the path of `config.yaml`.
Instead of applying your suggestion, how about it?

```bash
cat << EOF > $(directory name)/config.yaml
...
EOF
```

Reply:
I think it is better the specify the `config.yaml` path if we only need this file in the example.
```bash
% docker run -p 8081:8081 --rm -v $(pwd)/config.yaml:/etc/server/config.yaml -it vdaas/vald-agent-ngt
```

Reply:
@kevindiu 
I updated document, could you confirm it please?

Reply:
it is working.